sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -275,15 +275,15 @@ object ScalaJSPluginInternal {
         import ScalaJSOptimizer._
         val outCP = (scalaJSOptimizer in fastOptJS).value.optimizeCP(
             (scalaJSPreLinkClasspath in fastOptJS).value,
-            Config(
-                output = AtomicWritableFileVirtualJSFile(output),
-                cache = Some(taskCache),
-                wantSourceMap = (emitSourceMaps in fastOptJS).value,
-                relativizeSourceMapBase = relSourceMapBase,
-                bypassLinkingErrors = opts.bypassLinkingErrors,
-                checkIR = opts.checkScalaJSIR,
-                disableOptimizer = opts.disableOptimizer,
-                batchMode = opts.batchMode),
+            Config(AtomicWritableFileVirtualJSFile(output)).
+              withCache(Some(taskCache)).
```

Comment:
Our (undocumented) style is to put the `.` at the beginning of the next line rather than at the end of the previous line (with 2 spaces of indentation, as you did).

(this does not apply to .sbt files, where we follow sbt's style, which is to put the `.` at the end of the previous line)